pkg/config/cos.go

```diff
@@ -444,6 +431,92 @@ func updateBond(stage *yipSchema.Stage, name string, network *Network) error {
 	return nil
 }
 
+func updateBridge(stage *yipSchema.Stage, name string, mgmtNetwork *Network) error {
+	// add Bridge named MgmtInterfaceName and attach Bond named MgmtBondInterfaceName to bridge
+	var err error
+
+	needVlanInterface := false
+	// pvid is always 1, if vlan id is 1, it means untagged vlan.
+	if mgmtNetwork.VlanID >= 2 && mgmtNetwork.VlanID <= 4094 {
+		needVlanInterface = true
+	}
+
+	// setup pre up script
+	stage.Directories = append(stage.Directories, yipSchema.Directory{
+		Path:        "/etc/wicked/scripts",
+		Permissions: 0644,
+		Owner:       0,
+		Group:       0,
+	})
+
+	var preUpScript string
+	preUpScript, err = render("wicked-setup-bridge.sh", MgmtBondInterfaceName)
```

Comment:
prefer to use `:=` assignment if they are not top level variables.

Reply:
ok